src/components/PersonLink.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+
+interface Props {
+  person: Person
+  people: Person[],
+}
+
+export const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const mother = people.filter(mom => mom.name === person.motherName)[0];
+  const father = people.filter(dad => dad.name === person.fatherName)[0];
```

Comment:
better to use find method instead of filter, 'cause you want to find the element not filter the array